substrate/frame/staking-async/src/pallet/impls.rs

```diff
@@ -856,6 +861,129 @@ impl<T: Config> Pallet<T> {
 	pub fn api_pending_rewards(era: EraIndex, account: T::AccountId) -> bool {
 		Eras::<T>::pending_rewards(era, &account)
 	}
+
+	/// Calculate the total stake of the lowest portion validators and store it for the planned era.
+	///
+	/// Removes the stale entry from [`EraLowestRatioTotalStake`] if it exists.
+	pub(crate) fn calculate_lowest_total_stake() {
+		// Only calculate if unbonding queue params have been set.
+		if let Some(params) = UnbondingQueueParams::<T>::get() {
+			// Determine the total stake from the lowest portion of validators and persist for the
+			// era.
+			let mut exposures: Vec<_> =
+				ElectableStashes::<T>::get().into_iter().map(|(_, b)| b).collect();
+			let validators_to_check =
+				(params.lowest_ratio * exposures.len() as u32).max(1) as usize;
+
+			// Sort exposure total stake by the lowest first, and truncate to the lowest portion.
+			exposures.sort();
+			exposures.truncate(validators_to_check);
+
+			// Calculate the total stake of the lowest portion validators.
+			let total_stake = exposures.into_iter().sum();
+
+			// Store the total stake of the lowest portion validators for the planned era.
+			EraLowestRatioTotalStake::<T>::mutate(|lower_ratio| {
+				if lower_ratio.is_full() {
+					lower_ratio.remove(0);
+				}
+				lower_ratio.force_push(total_stake);
+			});
+		}
+	}
+
+	/// Gets the lowest of the lowest validator stake entries for the last upper bound eras.
+	pub fn get_min_lowest_stake() -> BalanceOf<T> {
+		// Find the minimum total stake of the lowest portion validators over the configured number
+		// of eras.
+		EraLowestRatioTotalStake::<T>::get().into_iter().min().unwrap_or(Zero::zero())
```

Comment:
```suggestion
		EraLowestRatioTotalStake::<T>::get().into_iter().min().unwrap_or(Bounded::max_value())
```

This is a better default, as it would lead to all unbonding times being the maximum, while this queue is not filled. 

Even better is to use an `Option` here -- if the past 28 eras are not filled, it should return `None`, which means we should not process any ubonding requests "faster".

Reply:
> This is a better default, as it would lead to all unbonding times being the maximum, while this queue is not filled.

Quite the opposite. This will increase `max_unstake` to its maximum, which will make any arbitrary amount to be unbonded in the minimum time.

The more stake in validators, the more a user can unbond without penalty, as the system has enough stake to slash.